.github/workflows/smoketest.yml

```diff
@@ -4,6 +4,12 @@ on:
   push:
   schedule:
     - cron: 0 0 * * *
+  pull_request:
+    branches:
+      - "**"
+
+env:
```

Comment:
Thank you - I only have just started migrating from travis so am new to github actions.